CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 
 # Changelog
 
+## Forthcoming
+
+MotoROS2 is now built against `micro_ros_motoplus` version `20230724`.
```

Comment:
Oh, this should read `20240710`. I'll update that after the binaries get tested.